packages/model/src/research-output.ts

```diff
@@ -170,7 +173,7 @@ export type ResearchOutputPostRequest = {
   link?: string;
 
   labs?: string[];
-  authors?: string[];
+  authors?: ExternalAuthorPostRequest[];
```

Comment:
quick question - how do we associate `Users` as authors? as you know the `ResearchOutputResponse` `authors` field contains a union of two types - shouldn't our input reflect it? I understand that at the moment it is impossible to link authors, we just link external authors?

https://github.com/yldio/asap-hub/blob/4d76557d1ab0de189274b2d651dfb03c3180ee2f/packages/model/src/research-output.ts#L160